wp-content/themes/currentorg/partials/projects-list-item.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/*
+ * The default template for displaying content
+ *
+ * @package Largo
+ */
+$args = array (
+	// post-specific, should probably not be filtered but may be useful
+	'post_id' => $post->ID,
+	'hero_class' => largo_hero_class( $post->ID, FALSE ),
+
+	// only used to determine the existence of a youtube_url
+	'values' => get_post_custom( $post->ID ),
+
+	// this should be filtered in the event of a term-specific archive
+	'featured' => false,
+
+	// $show_thumbnail does not control whether or not the thumbnail is displayed;
+	// it controls whether or not the thumbnail is displayed normally.
+	'show_thumbnail' => TRUE,
+	'show_excerpt' => TRUE,
+);
+
+$args = apply_filters( 'largo_content_partial_arguments', $args, get_queried_object() );
+
+extract( $args );
+
+$entry_classes = 'entry-content';
+
+$show_top_tag = largo_has_categories_or_tags();
+
+if ( $featured ) {
+	$entry_classes .= ' span10 with-hero';
+	$show_thumbnail = FALSE;
+}
+
+?>
+<article id="post-<?php the_ID(); ?>" <?php post_class('clearfix projects-list-item'); ?>>
+
+	<?php
+		// Special treatment for posts that are in the Homepage Featured prominence taxonomy term and have thumbnails
+		if ( $featured && ( has_post_thumbnail() || $values['youtube_url'] ) ) { ?>
+			<header>
+				<div class="hero span12 <?php echo $hero_class; ?>">
+				<?php
+					if( has_post_thumbnail() ){
+						echo( '<a href="' . get_permalink() . '" title="' . the_title_attribute( array( 'before' => __( 'Permalink to', 'largo' ) . ' ', 'echo' => false )) . '" rel="bookmark">' );
+						the_post_thumbnail( 'full' );
+						echo( '</a>' );
+					}
+				?>
+				</div>
+			</header>
+		<?php } // end Homepage Featured thumbnail block
+
+		echo '<div class="' . $entry_classes . '">';
+
+		if ( largo_has_categories_or_tags() ) {

```

Comment:
@benlk Do we need to write a customized function for this since we're using custom taxonomies instead of normal categories and tags? `largo_has_categories_or_tags()` returns `false` here.

Reply:
We'll also need to note that we need to add in not just the category tags here but also the project status, whether we do it now or when the final designs are done.

Reply:
We will, yes. Work in the branch for #122 renames this partial and starts work on a better layout for list items. https://github.com/INN/umbrella-currentorg/blob/122-search-and-filter/wp-content/themes/currentorg/partials/content-projects-list-item.php